.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: GitHub Actions Demo
+run-name: Pushed by ${{ github.actor }} 🚀
+on: [push]
+jobs:
+  test-xfuser:
+    name: Test xfuser
+    runs-on: [self-hosted, linux, x64]
+    strategy:
+      fail-fast: false
+      max-parallel: 5
+      matrix:
+        python-versions: [3.11]
+        torch-versions: [4.2.1]
+        include:
+          - python-versions: 3.11
+            python-versions-full: 3_11
+          - torch-versions: 4.2.1
```

Comment:
torch version 4.2.1? 
should it be 2.4.1?

Reply:
Sorry, it's a typo. I have fixed this issue in the latest commit (6c3ae7). And now the version name becomes 2.4.1.